cpp/src/binaryop/compiled/binary_ops.cu

```diff
@@ -295,13 +295,13 @@ std::unique_ptr<column> string_null_min_max(column_view const& lhs,
     *lhs_device_view, *rhs_device_view, op, output_type, lhs.size(), stream, mr);
 }
 
-void operator_dispatcher(mutable_column_device_view& out,
-                         column_device_view const& lhs,
-                         column_device_view const& rhs,
-                         bool is_lhs_scalar,
-                         bool is_rhs_scalar,
-                         binary_operator op,
-                         rmm::cuda_stream_view stream)
+__forceinline__ void operator_dispatcher(mutable_column_device_view& out,
```

Comment:
Have you tested with inline instead of force inline? We should prefer standard keywords wherever possible.

Reply:
I specifically requested this as the regression is caused by a lack of inlining due to crossing a threshold of complexity.

Reply:
Actually, this is the wrong place to put it because this is host code.

Reply:
I have moved the force_inline to the correct place and re-running the benchmarks.